wire-ios-notification-engine/Sources/NotificationSession.swift

```diff
@@ -337,6 +337,10 @@ public final class NotificationSession {
                 return
             }
 
+            if let clientId = ZMUser(context: self.coreDataStack.syncContext).selfClient()?.safeRemoteIdentifier.safeForLoggingDescription {
```

Comment:
Maybe it's easier to read in two parts?
```suggestion
            let selfClient = ZMUser(context: self.coreDataStack.syncContext).selfClient()
            if let clientId = selfClient?.safeRemoteIdentifier.safeForLoggingDescription {
```